Misc/NEWS.d/next/C_API/2025-06-25-01-03-10.gh-issue-135906.UBrCWq.rst

```diff
@@ -0,0 +1 @@
+Fix compilation errors when compiling the internal headers with a C++ compiler.
```

Comment:
compiler errors or compiler warnings?

Reply:
Compilation error

Reply:
But is it because you treat warnings as errors in your CI?

Reply:
`-Werror` is not used on the CI. So there are really compiler errors, ok ;-)